src/main/kotlin/io/sip3/salto/ce/sip/SipMessageHandler.kt

```diff
@@ -65,15 +72,15 @@ open class SipMessageHandler : AbstractVerticle() {
             config.getString("time-suffix")?.let {
                 timeSuffix = DateTimeFormatter.ofPattern(it)
             }
-
             config.getJsonObject("sip")?.getJsonObject("message")?.getJsonArray("exclusions")?.let {
                 exclusions = it.map(Any::toString).toSet()
             }
-
-            config.getJsonObject("sip")?.getJsonObject("message")?.getLong("check-udf-period")?.let {
+            config.getJsonObject("udf")?.getLong("check-period")?.let {
                 checkUdfPeriod = it
             }
-
+            config.getJsonObject("udf")?.getLong("execute-period")?.let {
```

Comment:
What is a final name for this parameter: `execute-timeout` or `execute-period`?

Reply:
fixed, copy-paste typo